src/cppgc_helpers.h

```diff
@@ -0,0 +1,57 @@
+#ifndef SRC_CPPGC_HELPERS_H_
+#define SRC_CPPGC_HELPERS_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include "cppgc/garbage-collected.h"
+#include "env.h"
+#include "memory_tracker.h"
+#include "v8-cppgc.h"
+#include "v8.h"
+
+namespace node {
+
+#define ASSIGN_OR_RETURN_UNWRAP_CPPGC(ptr, obj, ...)                           \
```

Comment:
Would you mind adding a small documentation/comment around what cppgc is, some important links etc to this document?

Reply:
That should probably go into `src/README.md`.